spec/models/order_creator_spec.rb

```diff
@@ -10,4 +10,13 @@
     expect(order.user_id).to eq(user.id)
   end
 
+  it "can calculate total price" do
+    item1 = Item.create(name: 'Soda', description: 'Sugary', price: 100, category_id: 1)
+    item2 = Item.create(name: 'Burger', description: 'beef', price: 200, category_id: 2)
```

Comment:
Line is too long. [88/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.